biblio.ref

```diff
@@ -6440,7 +6440,7 @@
 %L WINDOWS-GLYPH-PROC
 %T Windows Glyph Processing
 %A John Hudson
-%I Microsoft Typogrraphy
+%I Microsoft Typography
 %U http://www.microsoft.com/typography/developers/opentype/default.htm
```

Comment:
This also now redirects to https://docs.microsoft.com/en-us/typography/develop/processing-part1